components/CreateChallenge/FormGroup03ChallengeTime.tsx

```diff
@@ -57,6 +58,7 @@ function FormGroup03ChallengeTime({navigation, route}, props) {
           }
           placeholder=" 10 minutes"
```

Comment:
placeholer should be "10" because when I filled "10 minutes", that was converted NaN.